Sources/Public/Internal/CrashLoopDetector/BacktraceCrashLoopCounter.swift

```diff
@@ -0,0 +1,79 @@
+//
+//  BacktraceCrashLoopCounter.swift
+//  Backtrace
+//
+
+import Foundation
+
+@objc internal class BacktraceCrashLoopCounter: BacktraceCrashLoop {
```

Comment:
1. Do we reset the counter somewhere? Let's pretend we have 4 random crashes in the last month. Do we keep 4 as a counter all the time and we wait until the 5th crash happens?
2. What we do in a situation when the session ends with a crash but it wasn't a startup crash? Its a really popular use case - In this situation we have something that we can call "crash loop" from the code perspective, however, the developer shouldn't worry about this situation.

Reply:
1. For now counter resets when we detect a crash loop. Yes we keep 4 as a counter all the time. 
2. I didn't come with idea how to differ startup crashes right upon launch and crash somewhere after some time after launch. But comparing timestamps between didFinishLaunch and PLCR's crash signal handler maybe (though again - we have limited time here and we'll need to persist result - what brings back necessity to make a file write operation)

Reply:
There Is no counter any more